core/base/src/TROOT.cxx

```diff
@@ -2737,6 +2739,35 @@ void TROOT::SetMacroPath(const char *newpath)
       macroPath = newpath;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// If `web` is not empty it defines where web graphics should be rendered
+///
+/// - "cef" web graphics is rendered via the Chromium Embedded Framework
+/// - "qt5" web graphics is rendered via Qt5
+/// - "browser" or "" web graphics is rendered in the default web browser
+/// - "off" turn off the web display and come back to normal mode.
+///
+/// It also turn the normal graphics to Batch to avoid the loading of local
+/// graphics libraries.
+
+void TROOT::SetWebDisplay(const char *webdisplay)
+{
+   fWebDisplay = webdisplay;
+   fWebDisplay.ToLower();
+   if (fWebDisplay.Contains("cef")     || fWebDisplay.Contains("qt5") ||
+       fWebDisplay.Contains("browser") || fWebDisplay.Length() ==0) {
```

Comment:
@couet 
Olivier, fWebDisplay can have much more values and not only "cef" or "qt5" or "browser"
It can be name of browser or name of entry in rootrc file, where exact path to browser is written.

Reply:
@linev 
Hi Sergey, Should we add "Chrome" "safari" "IE" "firefox" ? I suggest we take it from rootrc when the option is left empty ? "browser" will be the default browser on the machine.

Reply:
@couet 
I do not like to limit allowed string values at this place.
It should be enough to set **any** string, which will be interpreted much later in TWebWindowManager. And one should be able to disable web mode when required.